.github/CODEOWNERS

```diff
@@ -76,4 +76,6 @@
 
 /src/connection-monitor-preview/ @haroldrandom
 
-/src/support/ @mit2nil @shahbj79
\ No newline at end of file
+/src/support/ @mit2nil @shahbj79
+
+/src/azext_synapse/ @idear1203 @aim-for-better
```

Comment:
/src/synapse/

Reply:
> /src/synapse/

fixed

Reply:
fixed

---
In reply to: [386994103](https://github.com/Azure/azure-cli-extensions/pull/1339#discussion_r386994103) [](ancestors = 386994103)